src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -0,0 +1,1278 @@
+<?php
+/**
+ * HTML API: WP_HTML_Processor class
+ *
+ * ### Breadcrumbs
```

Comment:
Is this headline missing some explanatory comment? 🙂

Reply:
Ah, looks like [it's explained in the class PHPDoc](https://github.com/WordPress/wordpress-develop/pull/4519/files?diff=unified&w=0#diff-f822c7309a0a11d3993b378b85ef4a60588af0afac714f7e1590130aa4e300e5R53-R91).

Reply:
oh I need to rip that out. still working through some of the docblocks